joblib/_parallel_backends.py

```diff
@@ -170,8 +170,8 @@ class AutoBatchingMixin(object):
     MAX_IDEAL_BATCH_DURATION = 2
 
     # Batching counters
-    _effective_batch_size = 1
-    _smoothed_batch_duration = 0.0
+    _effective_batch_size = _initial_batch_size = 1
```

Comment:
I thought I commented on this but I can not find it. I think `_effective_batch_size` should be an instance variable (initialized to `_initial_batch_size`) so remove it from here, same thing for `_smoothed_batch_duration`.

Reply:
And for consistency `_initial_effective_batch_size`.

Reply:
> I thought I commented on this but I can not find it

See [here](https://github.com/joblib/joblib/pull/556#pullrequestreview-66192825) in first outdated comment. I thought I was addressing your request with the actual change, since the `initial_` attributes are first defined with `_effective_batch_size` and `_smooth_batch_duration` and initialized with the same value.

Ok for renaming to `_initial_effective_batch_size`

Reply:
It's probably that you have done too much C++ and that my explanation is not that good ;-).

Basically 
```py
class A(object):
    variable = 1
```

is not the same as:

```py
class A(object):
    def __init__(self):
        self.a = 1
```

I want the latter for `_effective_batch_size` and `_smoothed_batch_duration` because I think that it does not make sense to have them available at the class level.

Reply:
I just pushed an update